cmd/kubelet/app/options/options.go

```diff
@@ -541,8 +541,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Int64Var(&c.MaxOpenFiles, "max-open-files", c.MaxOpenFiles, "Number of files that can be opened by Kubelet process.")
 
 	fs.StringVar(&c.ContentType, "kube-api-content-type", c.ContentType, "Content type of requests sent to apiserver.")
-	fs.Int32Var(&c.KubeAPIQPS, "kube-api-qps", c.KubeAPIQPS, "QPS to use while talking with kubernetes apiserver")
-	fs.Int32Var(&c.KubeAPIBurst, "kube-api-burst", c.KubeAPIBurst, "Burst to use while talking with kubernetes apiserver")
+	fs.Int32Var(&c.KubeAPIQPS, "kube-api-qps", c.KubeAPIQPS, "QPS to use while talking with kubernetes apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags")
```

Comment:
Mhmm I agree this flag could have a more descriptive name ... ideally `KubeApiserverQPS`. But I also agree that changing it is probably a whole ordeal.

One small nit - instead of indirectly stating that this flag only appplies to the apiserver by listing the apis it does not apply to, could we instead directly state this flag only applies to the apiserver?

My concern is this indirect method will be come confusing/false should we add more apis...

Reply:
@mattjmcnaughton 
> could we instead directly state this flag only applies to the apiserver

events and node heartbeat updates are also going to API server. It's just `kube-api-qps` parameter throttles every request **but** events and node heartbeat. Calling out the exceptions just seemed easier than whitelisting all the possible objects.

Reply:
Ah gotcha - so if a new API server were to be added later, it would be throttled by this parameter? If yes, than I agree with you :)